anchore_engine/services/policy_engine/engine/feeds/feeds.py

```diff
@@ -906,26 +907,84 @@ def _switch_active_grypedb(
         )
         db.add(self.grypedb_meta)
 
-    def _process_group_file_records(
+    @staticmethod
+    def _run_grypedb_sync_task(
```

Comment:
this method got carried over from a rebase. it was deleted in one of my PRs. you should remove it.

Reply:
Oh yeah good catch, this got lumped in with one of my changes when I resolved conflicts